lib/config/index.js

```diff
@@ -182,6 +204,15 @@ async function getConfig() {
     config.security.verifyRateLimit = Number(
```

Comment:
`getOptionalParameter` can return `undefined` which is not nice with the `Number` call

Reply:
This has been fixed in the latest commit.